Conditional Expressions/Greatest_Of_4_Numbers.py

```diff
@@ -0,0 +1,23 @@
+a = input ("Enter the first number: ")
+a = int (a)
+b = input ("Enter the second number: ")
+b = int (b)
+c = input ("Enter the third number: ") 
+c = int (c)
+d = input ("Enter the fourth number: ")
+d = int (d)
+
+if(a>d):

```

Comment:
Can you compare all the 4 variables altogether?

Reply:
Nope the question required it that way